src/README.md

```diff
@@ -196,6 +196,55 @@ int main(int argc, char* argv[]) {
 }
 ```
 
+### Performance Metrics
+
+`ov.genai.PerfMetrics` (referred to as `PerfMetrics` for simplicity) is a structure that holds performance metrics for each generate call. `PerfMetrics` hold fields with mean and standard deviations for the following metrics:
```

Comment:
There's
> `PerfMetrics` has

below. Like `PerfMetrics` is singular. But here
> `PerfMetrics` hold

`hold` doesn't have `s`, like if `PerfMetrics` were plural. Bot of the approaches seem to be valid, but you need to stick to one of them.

Reply:
agreed. Sticked to the plural